_posts/2019-03-26-technical-debt.md

```diff
@@ -0,0 +1,141 @@
+---
+title: Dług techniczny
+excerpt: Porównałem dług techniczny do długu finansowego — efektywne narzędzie do ominięcia braku zasobów w jednym momencie, licząc na to, że w późniejszym czasie będzie nam łatwiej „spłacić” takie zobowiązanie. Wspomniałem o nieoczywistych momentach, w których zaciągamy dług, jak np. podczas doboru nieodpowiedniej technologii, odchodzenie od rynkowych standardów albo... nic nie robiąc, bo dług techniczny sam z siebie narasta. Wspomniałem też o tym jak sobie z nim radzić w organizacji — przede wszystkim poprzez mierzenie jego poziomu oraz aktywną pracę z Klientem, aby utrzymywać go na komfortowym poziomie
+category: technology
+date: 2019-03-26 22:34
+author: puck
+layout: post
+---
+
+Słowem wstępu.
+
+Prowadziłem ostatnio retrospekcję jednego z projektów. Przeglądając notatki z sekcji „problemy” zaczął kształtować się jeden, bardzo jasny komunikat — mamy problem z długiem technicznym. Nie był to też odosobniony przypadek, a w dodatku objawiał się w tym projekcie w szczególny — oraz dość nieprzewidywany — sposób. Był to doskonały pretekst, żeby podsumować sobie czym jest dług techniczny, co wpływa na jego pogłębianie oraz jak nim zarządzać.
+
+## Ale dług jest techniczny czy technologiczny?
+
+Nie zbaczając za daleko z tematu: ja piszę o technicznym, jeśli ktoś życzy sobie dowiedzieć czegoś o technologicznym, to zapraszam w inne miejsce.
```

Comment:
A wyjaśnisz w jednym zdaniu jaka jest różnica między tymi pojęciami? Z mojego riserczu w googlu :p wynika, że można ich używać wymiennie (ja też tak zawsze o nich myślałam). Więc możliwe, że nie każdy czytelnik skuma, o co Ci chodzi.

Reply:
A jeżeli nie chcesz na to tracić miejsca, to może w ogóle pominąć ten akapit?

Reply:
eee, nie, skasuję ten fragment :)